### DISCUSSION TITLE
Corsair Theme

### DISCUSSION THREAD

**Discussion by MajorHeracles** [mathoudebine/turing-smart-screen-python#281] [Themes]
Bonjour à tous,

Je me suis amusé à créer un thème personnalisé inspiré de la marque Corsair que j'affectionne.
J'avais envie de vous le faire partager. ;)

Si vous en voulez d'autres dans le genre sur le thème Corsair, n'hésitez pas à me le demander !

![Capture d’écran 2023-07-08 131742](https://github.com/mathoudebine/turing-smart-screen-python/assets/138980478/2875f97a-a52f-41e4-b54c-ad43e97889ee)

[Corsair VIBES.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/11990890/Corsair.VIBES.zip)

**Comment by MajorHeracles** [comment]
Thème RAINBOW

![image](https://github.com/mathoudebine/turing-smart-screen-python/assets/138980478/ed3bc6f6-73b0-4f11-ac5d-c7802422c9d6)

[Corsair RAINBOW.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/11990886/Corsair.RAINBOW.zip)

**Comment by MajorHeracles** [comment]
Thème VERT

![image](https://github.com/mathoudebine/turing-smart-screen-python/assets/138980478/7d27b219-e439-4af5-9192-9638f29704f0)

[Corsair GREEN.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/11990883/Corsair.GREEN.zip)

**Comment by MajorHeracles** [comment]
Thème INDIGO

![image](https://github.com/mathoudebine/turing-smart-screen-python/assets/138980478/5b60c682-cb37-4b6a-88f2-b937fa7f4917)

[CORSAIR INDIGO.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/11991507/CORSAIR.INDIGO.zip)

**Comment by mathoudebine** [comment]
Salut, merci d'avoir partagé tes thèmes ici ! Ils sont top 👍

**Reply by DJRoby19** [reply]
Dear shame that you have never seen my themes

**Reply by mathoudebine** [reply]
Hey! Sorry I do not always check the discussion space and I do not receive notifications for new posts
I just checked and you made some really nice themes, thanks for that! I see a lot of people liked them and they are certainly using it daily so thanks for contributing!
